docs/product-user-guide/src/main/asciidoc/business-central-home-con.adoc

```diff
@@ -2,61 +2,162 @@
++
+This perspective is visible to users with the following roles:
+
+** Admin
+** Analyst
+** Developer
+
+* Dashboards
++
+The Dashboards perspective is the BAM component of {PRODUCT}. Use it to create custom perspectives for your projects. On the Dashboards perspective, you can create a new group for the *Main* menu and add the perspectives that you create to that group. This perspective is visible to users with the following roles:
+
+** Admin
+** Analyst
+** Developer
+
+.DevOps
+The DevOps group contains two perspectives:
+
+* Deployments
+
+[IMPORTANT]
+====
+The Deployments perspective is a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs), might
+not be functionally complete, and Red Hat does not recommend to use them for
+production. These features provide early access to upcoming product features,
+enabling customers to test functionality and provide feedback during the
+development process.
+For more information on Red Hat Technology Preview features support scope, 
+see https://access.redhat.com/support/offerings/techpreview/. 
+====
+
+The Deployments perspective enables you to provision projects on OpenShift. In this release, it is hidden by default. Perform the following steps to enable it:
+
+. Go to *Settings* → *Roles*.
+. Select the *admin* role.
+. In the Permissions section, click the arrow next to Perspectives.
+. Click the trash icon next to Deployments. The DevOps → *Deployments* link is now available on the Home page.
+
+* Execution Servers
+The Execution Servers perspective contains a list of the deployed Intelligent Process Server templates and containers associated with the templates. This perspective is visible to users with the following roles:
+
+** Admin
+** Developer
+
+.Manage
+The Manage group contains five perspectives:
+
+* Process Definitions
++
+The Process Definitions perspective contains a list of deployed process definitions. It enables you to instantiate and manage the deployed processes. This perspective is visible to users with the following roles:
+
+** Admin
+** Analyst
+** Developer
+** User
+** Process-admin
+
+* Process Instances
 +
-The view is available for users with the `admin` role only.
+The Process Instances perspective contains a list of the instantiated processes. It enables you to view their execution work flow and history. This perspective is visible to users with the following roles:
+
+** Admin
+** Analyst
+** Developer
+** User
+** Process-admin
+
+* Task Administration
 +
-** *Administration* perspective contains:
-*** The *File Explorer* view with available asset repositories.
-*** The editor area on the right of the *File Explorer* view, where the respective editor appears when a resource is opened.
+The Task Administration perspective contains a list of all manually created tasks, created by all users. This perspective is visible to users with the following roles:
```

Comment:
My previous comment still applies:

There's no way to create tasks "manually". At the moment the tasks are always instantiated when Human Task node of some running process definition is triggered. Perhaps: manually created tasks -> Human tasks from running process instances?

EDIT: I see you already applied that to "Tasks perspective" section, so the same should be applied here.